### DISCUSSION TITLE
看了md文件还是不清楚youtube源怎么使用大佬可以教教不

### DISCUSSION THREAD

**Discussion by nidemkzi** [1715173329/UnblockNeteaseMusic#293] [Q&A] [answered]
看了md文件还是不清楚youtube源怎么使用大佬可以教教不
// const proxy = require('url').parse('http://127.0.0.1:1080')
const proxy = undefined;
这两行我修改成
const proxy = require('url').parse('http://127.0.0.1:1080')
//const proxy = undefined;
依然不能使用

**Comment by nidemkzi** [comment]
刚看到有个-u参数加上去就可以了， 但是貌似只能http的代理， 我想使用sock5的代理还是不行

**Reply by pan93412** [reply]
目前只支援 HTTP proxy 而已

**Comment by nidemkzi** [comment, marked as answer]
另外大佬想问下。 我看cmd那边已经解析出来url了。 复制到浏览器打开也不会卡顿， 为啥在网易云这边要好久才加载出来，而且速度不快。是不是可以使用yt-download和youtube-dl这两个源？直接-o加上这两个参数并不能运行。。


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "UnblockNeteaseMusic/server"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年10月31日(星期天) 上午8:34
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [UnblockNeteaseMusic/server] 看了md文件还是不清楚youtube源怎么使用大佬可以教教不 (Discussion #293)





 
目前只支援 HTTP proxy 而已
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by pan93412** [reply]
我覺得是網易雲的問題。解析出來之後先按一下下一首，再按一下上一首應該就可以正常播放。

**Reply by pan93412** [reply]
`yt-download` 我記得好像壞掉了；`youtube-dl` 要先在你的電腦裝 `youtube-dl` 才能使用，解析速度或許會快一點：

```
pip install youtube-dl
```

**Comment by nidemkzi** [comment]
安装了依然无法使用耶。。






------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "UnblockNeteaseMusic/server"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年10月31日(星期天) 上午8:45
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [UnblockNeteaseMusic/server] 看了md文件还是不清楚youtube源怎么使用大佬可以教教不 (Discussion #293)





 
yt-download 我記得好像壞掉了；youtube-dl 要先在你的電腦裝 youtube-dl 才能使用，解析速度或許會快一點：
 pip install youtube-dl  
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by pan93412** [reply]
顯示什麼錯誤？直接執行 `youtube-dl` 呢？

**Comment by nidemkzi** [comment]
Please check the availability of match sources.找不到匹配的源， 有收到我发的截图吗


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "UnblockNeteaseMusic/server"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年10月31日(星期天) 上午8:56
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [UnblockNeteaseMusic/server] 看了md文件还是不清楚youtube源怎么使用大佬可以教教不 (Discussion #293)





 
顯示什麼錯誤？
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by pan93412** [reply]
沒收到截圖。應該是 `youtubedl` 才對，抱歉。

**Comment by nidemkzi** [comment]
噢 确实是这个， 我试试速度和-o youtube看看有没有区别 感谢~~








-----------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "UnblockNeteaseMusic/server"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年10月31日(星期天) 上午9:00
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [UnblockNeteaseMusic/server] 看了md文件还是不清楚youtube源怎么使用大佬可以教教不 (Discussion #293)





 
沒收到截圖。應該是 youtubedl 才對，抱歉。
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
